examples/14_blite_rc_car/14_blite_rc_car.ino

```diff
@@ -2,28 +2,17 @@
 #include "remote.h"
 #include <WebSocketsServer.h>
```

Comment:
Websocketserver  libraray is not required here, its already declared inside blite.